roxie/ccd/ccdserver.cpp

```diff
@@ -11224,7 +11225,7 @@ class CRoxieServerXmlWriteActivity : public CRoxieServerDiskWriteActivity
         CRoxieServerDiskWriteActivity::start(parentExtractSize, parentExtract, paused);
         OwnedRoxieString xmlpath(xmlHelper.getXmlIteratorPath());
         if (!xmlpath)
-            rowTag.set("Row");
+            rowTag.set(DEFAULTXMLHEADER);
```

Comment:
There are quite a few places like this where you are using DEFAULTXMLHEADER, but I think you meant DEFAULTXMLROWTAG?